src/structstore_pybind.cpp

```diff
@@ -257,6 +295,17 @@ void register_structstore_pybind(py::module_& m) {
         auto lock = list.read_lock();
         return to_object<false>(list[index].get_field());
     });
+    list.def("__delitem__", [](List& list, size_t index) {
+        auto lock = list.write_lock();
+        list.erase(index);
+    });
+    list.def("__iter__", [](List& list) {
+        auto lock = list.read_lock();
+        return pybind11::make_iterator(list.begin(), list.end());
+    }, py::keep_alive<0, 1>());
+    list.def("__eq__", [](List& l, List& o) {
+        return l == o;
```

Comment:
See comments above, I guess `&l == &o` might show more obviously what is going on